packages/react-hooks/src/mixer/useMixerInfo.ts

```diff
@@ -0,0 +1,44 @@
+import { useCall } from '@webb-dapp/react-hooks';
+import { MixerInfo, ScalarData } from '@webb-tools/types/interfaces';
+import { useMemo } from 'react';
+
+/**
+ * Class representing {MixerInfo} with a native js types
+ * */
+class MixerInfoWrapper {
+  constructor(private _inner?: MixerInfo) {}
+
+  get inner() {
+    return this._inner;
+  }
+
+  /**
+   * Tell wither  inner type exists or not
+   * */
+  get ready() {
+    return Boolean(this.inner);
+  }
+
+  get leaves(): ScalarData[] {
+    return this.inner?.leaves.toArray() ?? [];
+  }
+
+  get leaveU8a(): Uint8Array[] {
+    return this.leaves.map((leaf) => leaf.toU8a());
+  }
+}
+
+/**
+ * UseMixerInfos
+ *  @description   This will issue an RPC call to query.mixer.mixerGroups
+ *   @param {string} id which is the GroupId is optional if the is undefined the underlying rpc call won't take place
+ *
+ *
+ *  @return {MixerInfoWrapper}
+ * */
+export const useMixerInfo = (id?: string | undefined) => {
```

Comment:
return type missing

Reply:
This will be tackled on another PR since we are changing the MixerContext api